system/ee/ExpressionEngine/View/account/forgot_password.php

```diff
@@ -1,23 +1,27 @@
 <?php $this->extend('_templates/login'); ?>
 
 <div class="login__logo">
+    <?php if (ee('pro:Access')->hasRequiredLicense() && ee()->config->item('login_logo')) : ?>
+    <img src="<?=ee()->config->item('login_logo')?>" alt="Powered by ExpressionEngine&reg;">
```

Comment:
Should we be putting "Powered by ExpressionEngine" here?  I think the Site Name might be a more appropriate alt/title text to go along with the custom logo

Reply:
That makes sense. Changing here and also in other places.